src/odoc/bin/main.ml

```diff
@@ -69,8 +69,7 @@ let hidden =
   Arg.(value & flag & info ~docs ~doc ["hidden"])
 
 let dst ?create () =
-  let doc = "Output directory where the HTML tree is expected to be saved. \
-             Will be created if it doesn't exist." in
+  let doc = "Output directory where the HTML tree is expected to be saved." in
```

Comment:
Just a note: I think this description is inaccurate for the usage of `dst` in `compile-targets`. This inaccuracy is not introduced by these PRs, however.